src/chapters/intro-to-data/intro-to-data.rst

```diff
@@ -1,35 +1,34 @@
 Introduction To Data
 ====================
 
-Read the following articles, follow along where instructed:
-
 Methods and Techniques
 ----------------------
 
-1. `Data Analysis Methods & Techniques <https://www.datapine.com/blog/data-analysis-methods-and-techniques/>`__.
+Read the following article.
 
-* Key Takeaways: Video about qualitative vs quantitative data illustrates some key points!
+#. `Data Analysis Methods & Techniques <https://www.datapine.com/blog/data-analysis-methods-and-techniques/>`__.
 
 Data Analytics in Python
 ------------------------
 
-2. `A Beginner's Guide to Data Analysis in Python <https://towardsdatascience.com/a-beginners-guide-to-data-analysis-in-python-188706df5447>`__.
+Read the following article.
+
+#. `A Beginner's Guide to Data Analysis in Python <https://towardsdatascience.com/a-beginners-guide-to-data-analysis-in-python-188706df5447>`__.
 
-* Key Takeaways: How to take data from the computer and create a visualization using Python.
 
 Data Project Life Cycle
 -----------------------
 
-3. `Data project life cycle <https://www.northeastern.edu/graduate/blog/data-analysis-project-lifecycle/>`__.
+Read the following articles.
+
+#. `Data project life cycle <https://www.northeastern.edu/graduate/blog/data-analysis-project-lifecycle/>`__.  
 
-* Key Takeaways: Definitions of life cycles and Examples.
-  
-4. `Asking the right questions <https://www.datapine.com/blog/data-analysis-questions/%C2%A0%C2%A0>`__.
+#. `Asking the right questions <https://www.datapine.com/blog/data-analysis-questions/%C2%A0%C2%A0>`__.
 
 What is a Dataset?
 ------------------
 
-Think back to when you are working on a monthly budget. Let's say you have a singular spreadsheet 
+Think back to when you are working on a :ref:`monthly budget<monthly-budget>`. Let's say you have a singular spreadsheet 
```

Comment:
This paragraph is confusing and could use some reworking -> the original is less so.  Please remove the link as it doesn't link back and I had to try and find why you sent the reader to that page to begin with.